rbd/rbd_sparsify.go

```diff
@@ -0,0 +1,82 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package rbd
+
+/*
+#cgo LDFLAGS: -lrbd
+#include <errno.h>
```

Comment:
Double check: what was errno.h needed for?

Reply:
The test was complaining about no known kind of `C.EINVAL` if the `errno.h` header was not included.